text/0000-sbom.md

```diff
@@ -0,0 +1,180 @@
+# Meta
+[meta]: #meta
+- Name: Structured SBOMs
+- Start Date: 2021-06-02
+- Author(s): [@samj1912](https://github.com/samj1912), [@sophiewigmore](https://github.com/sophiewigmore), [@ForestEckhardt](https://github.com/ForestEckhardt)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes the following - 
+
+- Use of CycloneDX as the SBOM storage format.
+- Moving the SBOM from `<layer>.toml`, `launch.toml` and `build.toml` to separate files.
+
+# Definitions
+[definitions]: #definitions
+
+- SBOM/BOM: A software bill of materials (SBOM) is a list of components in a piece of software. Software vendors often create products by assembling open source and commercial software components. The SBOM describes the components in a product. In case of buildpacks the SBOM describes the contents of the various layers, buildpacks, stacks and the output app container.
```

Comment:
```suggestion
- SBOM (Software Bill Of Materials) / BOM: A list of components in a piece of software. Software vendors often create products by assembling open source and commercial software components. The SBOM describes the components in a product. In case of buildpacks the SBOM describes the contents of the various layers, buildpacks, stacks and the output app container.
```